README.md

```diff
@@ -199,14 +199,6 @@ You can pass parameters via standard `-D` syntax.
 
 `-Dfmt.skip` is whether the plugin should skip the operation.
 
-### Using with Java 16+ and Maven
-
-Since the JDK is more restrictive since version 16 you need to pass some [parameters](https://github.com/google/google-java-format#jdk-16) to the JVM to run the Google Java Formatter. To do so add the file  `.mvn/jvm.config` under your project's root directory with the following contents:
```

Comment:
This section is also removed upstream: https://github.com/google/google-java-format/commit/13ca73ebbfa86f6aca5f86be16e6829de6d5014c.